test/models/email_digests/new_developers_test.rb

```diff
@@ -10,6 +10,14 @@ class EmailDigests::NewDevelopersTest < ActionMailer::TestCase
     end
   end
 
+  test "do not send daily emails if no new developers signed up yesterday" do
+    create_developer(2.days.ago)
+
+    assert_no_emails do
+      EmailDigests::NewDevelopers.new.send_daily_digest
+    end
+  end
+
```

Comment:
Can this be done with the current seeds by time traveling to a specific date? Then we wouldn't have to manually create any developers or pull out the params from the other test.

Reply:
Done! 
What do you think?